q2_sample_classifier/visuals.py

```diff
@@ -102,8 +102,27 @@ def _linear_regress(actual, pred):
 def _plot_heatmap_from_confusion_matrix(cm, palette, vmin=None, vmax=None):
     palette = _custom_palettes()[palette]
     plt.figure()
-    return sns.heatmap(cm, vmin=vmin, vmax=vmax, cmap=palette,
-                       cbar_kws={'label': 'Proportion'})
+    scaletron, labelsize, dpi = 20, 8, 100
```

Comment:
As much as I love seeing "scaletron" here, that probably should've been removed in code when it originally was added to q2-feature-table.

```suggestion
    scaler, labelsize, dpi = 20, 8, 100
```